Jenkinsfile

```
@@ -112,7 +157,7 @@ pipeline {
                     dockerfile {
                         filename 'docker/Dockerfile.centos.7'
                         label 'docker_runner'
-                        additionalBuildArgs  '--build-arg UID=$(id -u)'
+                        additionalBuildArgs  '$BUILDARGS'
```

Can the function be used directly here?
```suggestion
                        additionalBuildArgs  docker_build_args()                       
```

If so, there is no need to set an env. var.